main.go

```diff
@@ -14,13 +17,40 @@ var (
 
 func main() {
 	app := cli.NewApp()
-	app.Name = "rancher-vxlan"
 	app.Version = VERSION
 	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "listen",
+			Value: ":8111",
+		},
 		cli.BoolFlag{
 			Name:   "debug, d",
 			EnvVar: "RANCHER_DEBUG",
 		},
+		cli.BoolFlag{
+			Name:   "disable-tx-offload",
+			EnvVar: "RANCHER_DISABLE_TX_OFFLOAD",
+		},
+		cli.StringFlag{
+			Name:   "physical-iface",
```

Comment:
I think we can derive this in code. Use one of the peers IP address to get the route and figure out the IP address.
Also we cannot make this static because in an environment we can have Ubuntu or Redhat or some other OS hosts, and they all have a different naming convention. Ex: eth0, ens2, etc ...